src/time-picker/clock.jsx

```diff
@@ -101,6 +106,16 @@ const Clock = React.createClass({
       container: {
         height: 280,
         padding: 10,
+        position: 'relative',
+      },
+
+      circle: {
+        position: 'absolute',
```

Comment:
The absolute position seems weird.
Can't we move this style to the `ClockHours` and `ClockMinutes` component?
*(IMHO those two components should be only one)*